src/app/app.component.ts

```diff
@@ -75,21 +109,23 @@ export class AppComponent {
    * Conditions loaded from TrialScope.
    */
   public trialScopeConditions: string[];
-  /*
-     variable for create object of search clinical trial request
-  * */
+  /**
+   * The Angular form model for search options.
+   */
   searchReqObject: { zipCode: string | null; travelRadius: string | null; phase: string; recruitmentStatus: string } = {
     zipCode: null,
     travelRadius: null,
     phase: 'any',
     recruitmentStatus: 'all',
   };
+  /*
+     variable for gathering patient bundle resources
+  * */
+  public bundleResources: any = [];
 
-  constructor(public commonService: CommonService, private spinner: NgxSpinnerService, private fhirService: ClientService, private convertService: ConvertCodesService) {
-    const paramPhase = '{ __type(name: "Phase") { enumValues { name } } }';
-    const recPhase = '{ __type(name: "RecruitmentStatusEnum") { enumValues { name } } }';
-    this.loadDropDownData(paramPhase, 'phase');
-    this.loadDropDownData(recPhase, 'rec');
+  constructor(private spinner: NgxSpinnerService, private trialScopeService: TrialScopeService, private fhirService: ClientService, private convertService: ConvertCodesService) {
+    this.loadDropDownData('Phase', 'phase');
+    this.loadDropDownData('RecruitmentStatusEnum', 'rec');
```

Comment:
This code is still, technically, TrialScope specific: it's loading the enumerated values of a TrialScope enum type via GraphQL.